genesyscloud/resource_genesyscloud_flow.go

```diff
@@ -213,38 +180,17 @@ func updateFlow(ctx context.Context, d *schema.ResourceData, meta interface{}) d
 	sdkConfig := meta.(*providerMeta).ClientConfig
```

Comment:
Looks like the updateFlow function is very similar to the createFlow function. See if you can try to roll them into one. Have a look at `createGroupRoles` or `createIdpAdfs` for an example of this.